docs/zh/UserGuide/Server/Config Manual.md

```diff
@@ -602,23 +593,6 @@
 |默认值|FLOAT |
 |改后生效方式|重启服务器生效|
 
-* enable\_partition
-
-|名字| enable\_partition |
-|:---:|:---|
-|描述| 是否启用数据分区。如果禁用，则所有数据都属于分区0 |
-|取值| BOOLEAN |
-|默认值| false |
-|改后生效方式|重启服务器生效|
-
-* partition\_interval
-
-|名字| partition\_interval |
-|:---:|:---|
-|描述| 对每个存储组内的数据进行分区的时间范围，单位为秒|
-|取值| LONG |
-|默认值|604800 |
-|改后生效方式|重启服务器生效|
```

Comment:
Why do you remove these? I think these parameters are meaningful....

Reply:
They are repeated, the above already contains；
![image](https://user-images.githubusercontent.com/18386653/108589280-ad9e7b80-7398-11eb-95ef-70ce6a913649.png)

Reply:
Maybe I was wrong. I just found out that they belong to different modules. I am not familiar with them yet. Please confirm, thank you!
![image](https://user-images.githubusercontent.com/18386653/108589583-edb22e00-7399-11eb-9061-a54e727dd6b1.png)
![image](https://user-images.githubusercontent.com/18386653/108589623-23efad80-739a-11eb-8b6b-fa391f094cf0.png)

Reply:
Ok, it makes sense. Thanks.